### PR TITLE
feat: add OpenAPI Preview (#517)

### PR DESCRIPTION
**Pull request by AndrewWalsh**
Add OpenAPI Preview.

Addresses 1/2 of https://github.com/apisyouwonthate/openapi.tools/issues/517.

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1700,6 +1700,16 @@
   v2: true
   v3: true
 
+- name: OpenAPI Preview
+  category: documentation
+  language: TypeScript
+  link: https://marketplace.visualstudio.com/items?itemName=zoellner.openapi-preview
+  github: https://github.com/zoellner/openapi-preview
+  description: View OpenAPI 3.1 specifications in Visual Studio Code using Spotlight Elements.
```
Comment (AndrewWalsh) [inline _data/tools.yml:1708]:
```suggestion
  description: Preview OpenAPI 3.1 specifications in Visual Studio Code using Spotlight Elements.
```

File: _data/tools.yml [hunk 2]
```diff
@@ -1700,6 +1700,16 @@
   v2: true
   v3: true
 
+- name: OpenAPI Preview
```
Comment (philsturgeon) [inline _data/tools.yml:1703]:
```suggestion
- name: VSCode OpenAPI Preview
```

File: _data/tools.yml [hunk 3]
```diff
@@ -1700,6 +1700,16 @@
   v2: true
   v3: true
 
+- name: VSCode OpenAPI Preview
+  category: documentation
+  language: TypeScript
+  link: https://marketplace.visualstudio.com/items?itemName=zoellner.openapi-preview
+  github: https://github.com/zoellner/openapi-preview
+  description: Preview OpenAPI 3.1 specifications in Visual Studio Code using Spotlight Elements.
```
Comment (philsturgeon) [inline _data/tools.yml:1708]:
```suggestion
  description: Preview OpenAPI v3.0 & v3.1 documents in Visual Studio Code using Spotlight Elements.
```

**Review by philsturgeon — APPROVED** [review]
Thank you!

**Comment by AndrewWalsh** [comment]
@philsturgeon no worries, I've still quite a bit to learn so I'll hold off on closing issues you raise without prior review.

**Comment by philsturgeon** [comment]
You're doing great!

**Pull request merged**